pkgs/tools/security/doas-sudo-shim/default.nix

```
@@ -0,0 +1,84 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, runCommand
+, asciidoctor
+, coreutils
+, gawk
+, glibc
+, util-linux
+, bash
+, gnugrep
+, makeBinaryWrapper
+, testers
+, doas-sudo-shim
+}:
+
+stdenv.mkDerivation rec {
+  pname = "doas-sudo-shim";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "jirutka";
+    repo = "doas-sudo-shim";
+    rev = "v${version}";
+    sha256 = "QYVqGxeWC7Tiz8aNY/LukwG4EW0km/RunGEfkzY/A38=";
+  };
+
+  nativeBuildInputs = [ asciidoctor makeBinaryWrapper ];
+  buildInputs = [ bash coreutils gawk glibc util-linux ];
+  checkInputs = [ gnugrep ];
```

```suggestion
  nativeBuildInputs = [ asciidoctor makeBinaryWrapper ];
  buildInputs = [ bash coreutils gawk glibc util-linux ];
```
thats already in stdenv

---

Done